packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -95,12 +95,15 @@ import ValidateRecipientIntro, {
 } from 'src/send/ValidateRecipientIntro'
 import SetClock from 'src/set-clock/SetClock'
 import TransactionReview from 'src/transactions/TransactionReview'
+import Logger from 'src/utils/Logger'
```

Comment:
this change seems unrelated. probably just a leftover right?

Reply:
It was a leftover, but I figured it's a useful log to have regardless (I was looking for something like this in the past), so I left it on purpose.